compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangNodeBuilder.java

```diff
@@ -2457,7 +2457,8 @@ public BLangNode transform(TemplateExpressionNode expressionNode) {
             case RAW_TEMPLATE_EXPRESSION:
                 return createRawTemplateLiteral(expressionNode.content(), getPosition(expressionNode));
             case REGEX_TEMPLATE_EXPRESSION:
-                return createRegExpTemplateLiteral(expressionNode.content(), getPosition(expressionNode));
+                return createRegExpTemplateLiteral(expressionNode.content(), getPosition(expressionNode),
+                        expressionNode.startBacktick(), expressionNode.endBacktick());
```

Comment:
```suggestion
                return createRegExpTemplateLiteral(expressionNode.content(), getPosition(expressionNode),
                        getPosition(expressionNode.startBacktick(), expressionNode.endBacktick()));
```

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/39466/commits/7ae121096651ef8776fb470d5f56096e846c08e6.